core/src/main/java/io/smallrye/openapi/runtime/scanner/dataobject/AnnotationTargetProcessor.java

```diff
@@ -169,9 +169,11 @@ Schema processField() {
             fieldSchema = MergeUtil.mergeObjects(new SchemaImpl(), typeSchema);
         }
 
-        for (AnnotationTarget contraintTarget : typeResolver.getConstraintTargets()) {
-            // Apply constraints from all bean properties associated with the schema property
-            BeanValidationScanner.applyConstraints(contraintTarget, fieldSchema, propertyKey, this);
+        if (context.getBeanValidationScanner().isPresent()) {
+            for (AnnotationTarget contraintTarget : typeResolver.getConstraintTargets()) {
+                // Apply constraints from all bean properties associated with the schema property
+                context.getBeanValidationScanner().get().applyConstraints(contraintTarget, fieldSchema, propertyKey, this);
+            }
         }
```

Comment:
For consistency with the other calls for BV scanning.
```suggestion
        context.getBeanValidationScanner().ifPresent(s -> {
            for (AnnotationTarget contraintTarget : typeResolver.getConstraintTargets()) {
                // Apply constraints from all bean properties associated with the schema property
                s.applyConstraints(contraintTarget, fieldSchema, propertyKey, this);
            }
        });
```